plugins/polio/js/src/domains/Budget/BudgetProcess/CreateBudgetProcessModal.tsx

```diff
@@ -86,31 +89,39 @@ const CreateBudgetProcessModal: FunctionComponent<Props> = ({
                     confirmMessage={MESSAGES.save}
                     allowConfirm={allowConfirm}
                     cancelMessage={MESSAGES.cancel}
+                    maxWidth="md"
                 >
-                    <Box mb={2} mt={2}>
-                        <Field
-                            label={formatMessage(MESSAGES.labelCountry)}
-                            name="country"
-                            component={SingleSelect}
-                            options={dropdownsData?.countries || []}
-                        />
-                    </Box>
-                    <Box mb={2}>
-                        <Field
-                            label={formatMessage(MESSAGES.labelCampaign)}
-                            name="campaign"
-                            component={SingleSelect}
-                            options={currentCampaignOptions}
-                        />
-                    </Box>
                     <Box mb={2}>
-                        <Field
-                            label={formatMessage(MESSAGES.labelRound)}
-                            name="rounds"
-                            component={MultiSelect}
-                            options={currentRoundsOptions}
-                        />
+                        <Divider />
                     </Box>
+                    <Grid container direction="row" item spacing={2}>
+                        <Grid item xs={6}>
+                            <Box mb={2}>
+                                <Field
+                                    label={formatMessage(MESSAGES.labelCountry)}
+                                    name="country"
+                                    component={SingleSelect}
+                                    options={dropdownsData?.countries || []}
+                                />
+                            </Box>
+                            <Field
+                                label={formatMessage(MESSAGES.labelCampaign)}
+                                name="campaign"
+                                component={SingleSelect}
+                                options={currentCampaignOptions}
+                            />
+                        </Grid>
+                        <Grid item xs={6}>
+                            <Field
+                                label={formatMessage(MESSAGES.labelRound)}
+                                name="rounds"
+                                component={MultiSelect}
+                                options={currentRoundsOptions}
+                                returnFullObject
```

Comment:
I managed to crash the modal while playing with rounds. I think it maybe when you have this prop to `true` and the value is `null`